artifacts/testing/C01-device-errors.feature

```diff
@@ -0,0 +1,118 @@
+Feature: CAMARA Common Artifact C01 - Test scenarios for device errors
+
+    Common error scenarios for POST operations with device as input either in the request
+    body or implied from the access.
+
+    Artifact parameters (to be replaced by values according to the API operation):
+    
+    - {{feature_identifier}} 
+
+    This is not a complete feature but a collection of scenarios that can be applied
+    with minor modifications to test plans.
+
+    These scenarios assume that other properties not explicitly mentioned in the scenario
+    are set by default to a valid value. This can be specified in the feature Background.
+

```

Comment:
We can have more clearer feature description to let know what should be removed or replaced  when using the templates. Mark the block to be removed when instantiating this feature file. Eg. $Hint_Start --to be removed { } $Hint_End .

Reply:
@jlurien  Please clarify in more better way

Reply:
@mdomale, please review the new wording. Feel free to add any suggestions